src/themes/elife/test/eifeDataProvider.test.ts

```diff
@@ -0,0 +1,59 @@
+import dataProvider from '../eLifeDataProvider'
+
+interface Response {
+  ok: boolean
+  json: Function
+}
+
+describe('eLife Data Provider ', () => {
+  describe('successfully querying a valid article id', () => {
+    it('does not throw', async () => {
+      const fetchMock = (): Promise<Response> =>
+        Promise.resolve({ ok: true, json: () => Promise.resolve() })
+      await expect(
+        dataProvider.query('validArticleId', fetchMock)
+      ).resolves.not.toThrow()
+    })
+
+    it('it exposes the URI of the article PDF', async () => {
+      const fetchMock = (): Promise<Response> =>
+        Promise.resolve({
+          ok: true,
+          json: () => Promise.resolve({ pdf: 'path-to-the.pdf' })
+        })
+      await expect(dataProvider.query('someId', fetchMock)).resolves.toEqual({
```

Comment:
Tried using `toContainEqual` here in order to the check for only the piece of data we care about in this specific test, but that kept failing unexpectedly. Also at https://github.com/stencila/thema/pull/111/files#diff-25876d902a7b9a110d8b13aa7d1d3af9R39